pyproject.toml

```
@@ -221,7 +221,7 @@ filterwarnings = [
   "ignore:'mode' parameter is deprecated and will be removed in Pillow 13:DeprecationWarning", # matplotlib toolbar icon creation with newer Pillow
   'ignore:.*A NumPy version .* is required for this version of SciPy.*:UserWarning',
   'ignore:.*Given trait value dtype "float64":UserWarning', # bogus numpy ABI warning (see numpy/#432)
-  'ignore:.*PYVISTA_USERDATA_PATH is not a valid directory.*:UserWarning',
+  'ignore:.*PYVISTA_VTK_DATA is not a valid directory.*:UserWarning',
```

Need to fix regex for multiline as this filter was broken by #7822, see https://github.com/pyvista/pyvista/pull/7822#issuecomment-3191965627
```suggestion
  'ignore:(?s).*PYVISTA_VTK_DATA is not a valid directory.*:\n.*:UserWarning',
```